### PR TITLE
MTP-2178 adjust header for mobile

### PR DESCRIPTION
**Pull request by petterkoding**
- add slightly larger search icon
- align items in header
- make logo smaller on mobile

logo is passed to Dropdown component in Web

see:  https://github.com/Mattilsynet/mattilsynet-web/pull/843

### CODE REVIEW

File: mt-kit/core/css/src/utilities/_layout.scss [hunk 1]
```diff
@@ -260,6 +260,10 @@ Only use if it does NOT affect accessibility
   grid-template-columns: 1fr max-content max-content max-content;
   align-items: center;
 
+  >.mt-link {
```
Comment (lha047) [inline mt-kit/core/css/src/utilities/_layout.scss:263]: Can you update the headers in the stories so we can see how this works?

**Review by lha047 — CHANGES_REQUESTED** [review]
Have a look at comments. 

Could you also fix the no-js styling?
Screenshot 2024-03-20 at 07 53 39

**Review by lha047 — CHANGES_REQUESTED** [review]
Good work :)

Did you look at the styling of the header without JS?

**Comment by petterkoding** [comment]
> Good work :)
> 
> Did you look at the styling of the header without JS?

See the other PR for styling without JS

**Review by lha047 — APPROVED** [review]
👍

**Pull request merged**